tests/tests.py

```diff
@@ -246,6 +246,10 @@ def test_rnasep_cm_database(self):
         """Check RNAse P covariance models."""
         self.verify_cm_database(config.RNASEP_CM_LIBRARY, 25)
 
+    def test_tmrna_cm_database(self):
+        """Check tmRNA covariance models."""
+        self.verify_cm_database(config.TMRNA_CM_LIBRARY, 5)
+
```

Comment:
Would it be possible to add a test similar to the [TestRnasep test case](https://github.com/r2dt-bio/R2DT/blob/3b44747ec48e6bf0edb2ba31bce4e1cb209b2f3a/tests/tests.py#L458)? Then I would be able to run the tests and if they all pass then the code can be merged.

Reply:
Added

Reply:
@nawrockie Thank you Eric! That was fast! 

If I run `just test Tmrna` it fails because `hits.txt` is different in the `tests/examples/` and `tests/results` folders. It seems to have something to do with the names of the sequences. Is this test passing for you?

Reply:
It worked, thank you Eric! 🎉